# the first door
target1_x : 2.5
target1_y : 0
# ar
target2_x : 3.6
target2_y : 0

target2_3_x : 2.5 
target2_3_y : 0

target3_x : 1.8
target3_y : 1.55

target3_4_x : 2.5
target3_4_y : 1.55

target4_x : 3.6
target4_y : 1.55

target4_5_x : 2.5
target4_5_y : 3.5

# between 4 and 5
tmp_target_x : 2.5
tmp_target_y : 1.55

# the last ar
target5_x : 3.55
target5_y : 3.5

# the second door
target6_x : 2.7
target6_y : 3.16

# behind the door
target7_x : 0
target7_y : 3.16

# land
land_target_x : 0
land_target_y : 3.1 # 3.1 2.3 1.5

# land ar_id
land_marker_id : 7 # 7 5 3

# ar track speed
track_speed : 0.08

# ar_track_err_max
ar_track_err_max : 0.08

# cross door pos_x
first_track_pos_x : 3.0
second_track_pos_x : -2.7

# fly height
fly_height : 0.75

# hit target time
pwm_time : 1.0


